### DISCUSSION TITLE
[HELP]How to  initialize "WidgetSlider"

### DISCUSSION THREAD

**Discussion by Panda98277** [maruohon/malilib#131] [Mod Development]
![image](https://github.com/maruohon/malilib/assets/118618400/45f721eb-8439-446c-bd61-9497d805b069)

**Comment by maruohon** [comment]
You give it the callback.

See here for an example usage: https://github.com/maruohon/malilib/blob/e2499660dfda575b0b4cd3bd9c69124a3148ee75/src/main/java/fi/dy/masa/malilib/gui/widgets/WidgetConfigOption.java#L348-L366

**Comment by Panda98277** [comment]
then how do i initalize "IConfigSlider",it's an an abstract class

**Reply by Panda98277** [reply]
i've done it
thank you
